code/game/machinery/newscaster/newscaster_machine.dm

```diff
@@ -432,6 +432,7 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/newscaster, 30)
 			bounty_value = text2num(params["bountyval"])
 			if(!bounty_value)
 				bounty_value = 1
+			bounty_value = clamp(bounty_value, 1, 1000)
```

Comment:
Should probably mention that you are limiting bounty value to a maximum of 1000 as well

Reply:
The JS ui already caps it at 1000, there was just no clamp applied to the beyond end.